app/services/code_climate/projects_summary.rb

```diff
@@ -21,8 +21,10 @@ def projects_rated_with(letter)
       ratings&.fetch(letter.to_s, 0)
     end
 
-    def rating_names
-      ratings.keys
+    def project_rates(&block)
+      ratings.each_pair(&block)
```

Comment:
Shouldn't we check if a block is given? Otherwise what would happen here?

Reply:
Yes, the name is misleading since it's an iterator. I will rename it to each_project_rate. Thanks!